umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-entire-umh.md

```diff
@@ -0,0 +1,123 @@
++++
+title = "Backup and Restore the United Manufacturing Hub"
+menuTitle = "Backup and Restore the UMH"
+description = "This page describes how to backup and restore the entire UMH."
+weight = 1000
++++
+
+This page describes how to back up the following:
+ - All Node-RED flows
+ - All Grafana dashboards
+ - The current Helm values for the united-manufacturing-hub chart
+ - All timescale tables used by the united-manufacturing-hub chart
+
+It does **not** back up:
+ - Other grafana data like Alerts, Users, ...
+ - Timescale data unrelated to the united-manufacturing-hub chart 
+ - Any other data in the cluster 
+ - TimescaleDB continuous aggregates: If you want to back them up, please follow [this guide](https://docs.timescale.com/self-hosted/latest/migration/schema-then-data/#recreate-continuous-aggregates)
+ - TimescaleDB policies: If you want to back them up, please follow [this guide](https://docs.timescale.com/self-hosted/latest/migration/schema-then-data/#recreate-policies)
+    
+
+## Requirements
+
+1. Stop stateful services -> kafka/redpanda and HiveMQ to prevent inconsistencies in the dataset.
+
+   ![Untitled](/images/production-guide/backup_recovery/backup-entire-umh/lensStopStatefulServices.png?width=75%)
+2. Create a Grafana API Token for an admin user:
+   - login to Grafana
+   - click on **settings** -> **API keys**
+   
+     ![Untitled](/images/production-guide/backup_recovery/backup-entire-umh/grafanaSettingsApiKey.png?)
+   - create a new API-key and change its role to **Admin**.
+   
+     ![Untitled](/images/production-guide/backup_recovery/backup-entire-umh/ApiKeycahngeRole.png?)
+   
+3. Download the [backup](https://github.com/united-manufacturing-hub/backup) repository. 
+4. Make sure you have postgresql installed.
+
+
+
+## Backup
+
+1. Open a Terminal inside the repository folder.
```

Comment:
Open a terminal and navigate to the folder containing the script

_give the command_